content/blog/adoptium-reproducible-builds/index.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Reproducible Builds at Eclipse Adoptium
+date: "2022-03-31T12:00:00+00:00"
+author: anleonar@redhat.com
+description: Eclipse Adoptium is working towards fully reproducible builds, this blog explains why
+ they are needed and the work ongoing to achieve them.
+tags:
+  - Temurin
+---
+
+## Why do we need Reproducible Builds?
+
+### Confidence in the high quality of the process to create the binaries
+
+Being able to repeatably build the same OpenJDK source in an identical manner, producing
+identical JDK binaries, shows confidence and quality in the production process.
+The necessity for producing identical results is to know exactly what is going into the binary and the exact repeatability of the
+build process, through what source and dependencies are used, to what compiler tooling is used.
+The fact that a reproducible identical JDK binary can only be achieved by all these stringent controls,
+shows the high quality of the processes used to bring the JDK binaries to the consumers.
+
+### Secure Supply Chain
+
+The reproduciblity of builds goes hand in hand with knowing exactly what goes into it, in other words 
+the Software Bill of Materials (SBOM). This detailed knowledge means we know the source and provenance
```

Comment:
One thing I forgot in my last review - perhaps include a list of the sort of things that are included in an SBOM for people who have never heard of it before (or an external reference if that's more applicable, and/or a link to one of ours :-) )